NAMESPACE

```diff
@@ -7,6 +7,7 @@ export(concat)
 export(dev_suppress)
 export(eval_code)
 export(get_code)
+export(get_env)
```

Comment:
Please add news entry. It's a prominent change

Reply:
cc8bcc227bc2a6a804974aec90e4bebcf8588c45